integration/service/create_test.go

```diff
@@ -309,6 +311,101 @@ func TestCreateServiceConfigFileMode(t *testing.T) {
 	assert.NilError(t, err)
 }
 
+// TestServiceCreateSysctls tests that a service created with sysctl options in
```

Comment:
Shouldn't we have a similar test for updates too?

Reply:
there isn't one for the other things tested here.